### PR TITLE
ACMS-3281: Refactor Drupal recommended settings.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #[ACMS-3281](https://acquia.atlassian.net/browse/ACMS-3281)

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

**Review by chandan-singh7929 — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]

**Comment by chandan-singh7929** [comment]
Note: Currently it will not allow to add along with blt, hence I think we need to remove that conflict part from composer.json

**Comment by rajeshreeputra** [comment]
> Note: Currently it will not allow to add along with blt, hence I think we need to remove that conflict part from composer.json

Please use [ACMS-3281](https://github.com/acquia/drupal-recommended-settings/tree/ACMS-3281) branch.

File: README.md [hunk 1]
```diff
@@ -44,17 +45,59 @@ require DRUPAL_ROOT . "/../vendor/acquia/blt/settings/blt.settings.php";
...
+// Mysql port.
+      'port' => '3306',
+    ],
+  ],
+];
+
+// Call generate method with database details.
+$settings->generate($dbSpec);
+```
+
```
Comment (chandan-singh7929) [inline README.md:91]: Why this is added in readme?

**Review by rajeshreeputra — COMMENTED** [review]
LGTM!

**Review by apathak18 — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]

**Pull request merged**